Script.php

```diff
@@ -682,50 +663,48 @@ function checksize($file)
  
  
  
- 
- 
- // ############################### Submit Assignment ##################################
-    if (!empty($_POST["frm_submitlab"])) {
+// ############################### Submit Assignment ##################################
+if (!empty($_POST["frm_submitlab"])) {
         
      
         
-     $lab_id=mysqli_real_escape_string($con,$_POST["lab_id"]);
-     $student_id=$_POST["student_id"];
-        $group_id=$_POST["group_id"];
-     
+  $lab_id=mysqli_real_escape_string($con,$_POST["lab_id"]);
+  $student_id=$_POST["student_id"];
+     $group_id=$_POST["group_id"];
+  
 
-       $instructions=mysqli_real_escape_string($con,$_POST["instructions"]);
-       $title=mysqli_real_escape_string($con,$_POST["title"]);
-       
-         $url=mysqli_real_escape_string($con,$_POST["url"]);
-       
-       
-       $deadline=$deadlinedate." ".$deadlinetime;
-      $date=  date("Y-m-d H:i");
-            
-       
-       
- // GET UPLOADED FILES
-       
-     
+    $instructions=mysqli_real_escape_string($con,$_POST["instructions"]);
+    $title=mysqli_real_escape_string($con,$_POST["title"]);
+    
+      $url=mysqli_real_escape_string($con,$_POST["url"]);
+    
+    
+    $deadline=$deadlinedate." ".$deadlinetime;
+   $date=  date("Y-m-d H:i");
+         
+    
+    
+// GET UPLOADED FILES
+    
+  
 
-         $labName = mysqli_query($con,"SELECT Title FROM `lab_reports_table` WHERE Lab_Report_ID=$lab_id");
-     while($row = mysqli_fetch_assoc($labName)) {$lab_name=$row['Title'];} 
+      $labName = mysqli_query($con,"SELECT Title FROM `lab_reports_table` WHERE Lab_Report_ID=$lab_id");
+  while($row = mysqli_fetch_assoc($labName)) {$lab_name=$row['Title'];} 
 
 
-       $target_dir =Create_dir("Lab_Report_Submisions/".$student_id."/".$lab_name."/");
-	   
-	   
-	   
-         $targetfile = $target_dir.$_FILES['attachment1']['name'];
-           $targetfile2 = $target_dir.$_FILES['attachment2']['name'];
-             $targetfile3 = $target_dir.$_FILES['attachment3']['name'];
-              $targetfile4 = $target_dir.$_FILES['attachment4']['name'];
-             
+    $target_dir =Create_dir("Lab_Report_Submisions/".$student_id."/".$lab_name."/");
+  
+  
+  
+      $targetfile = $target_dir.$_FILES['attachment1']['name'];
+        $targetfile2 = $target_dir.$_FILES['attachment2']['name'];
+          $targetfile3 = $target_dir.$_FILES['attachment3']['name'];
+           $targetfile4 = $target_dir.$_FILES['attachment4']['name'];
```

Comment:
@Teecloudy 

What if the instructor does not have four attachments?  I hope the program won't report **Key Not Found** error.

**Hui**

Reply:
Since this was the original code i think it wont bring any change. The student and instructors can upload upto 4 attachachments but the database can receive minimum 0 for instructor and minimum1 for student (Max being 4).